### PR TITLE
Fixed some minor issues

### PR DESCRIPTION
**Pull request by AAAstorga**

### CODE REVIEW

File: TCAT/Controllers/HomeViewController.swift [hunk 1]
```diff
@@ -83,6 +85,8 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
         let header = view as! UITableViewHeaderFooterView
         header.textLabel?.textColor = .secondaryTextColor
         header.textLabel?.font = tctSectionHeaderFont()
+        header.backgroundColor = .blue
```
Comment (AAAstorga) [inline TCAT/Controllers/HomeViewController.swift]: Will remove this

File: TCAT/Controllers/HomeViewController.swift [hunk 2]
```diff
@@ -83,6 +85,8 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
...
+        header.tintColor = .blue
```
Comment (AAAstorga) [inline TCAT/Controllers/HomeViewController.swift]: will remove this

File: TCAT/Controllers/HomeViewController.swift [hunk 3]
```diff
@@ -52,13 +51,15 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
         tableView.register(SearchResultsCell.self, forCellReuseIdentifier: "searchResults")
         tableView.register(SearchResultsCell.self, forCellReuseIdentifier: "cornellDestinations")
         view.addSubview(tableView)
+        view.setNeedsLayout()
```
Comment (myo3) [inline TCAT/Controllers/HomeViewController.swift]: not needed

File: TCAT/Controllers/SearchResultsTableViewController.swift [hunk 4]
```diff
@@ -54,6 +54,7 @@ class SearchResultsTableViewController: UITableViewController, UISearchResultsUp
         isRecentLocationsEmpty = recentLocations.isEmpty
         sectionExtraIndex = !isRecentLocationsEmpty ? 1 : 0
         formatSections()
+        definesPresentationContext = true
```
Comment (myo3) [inline TCAT/Controllers/SearchResultsTableViewController.swift]: check if can get rid of it

**Review by myo3 — APPROVED** [review]

**Pull request merged**